components/projects/Projects.tsx

```diff
@@ -0,0 +1,63 @@
+import { Grid } from "@mantine/core";
+import { useEthers } from "@usedapp/core";
+import { useEffect, useState } from "react";
+import { getManagerContract } from "../../utils/contracts";
+import LoadingCard from "./LoadingCard";
+import ProjectCard from "./ProjectCard";
+
+const Projects = () => {
+  const { library, chainId } = useEthers();
+  const [numProjects, setNumProjects] = useState<number>();
+  const [projects, setProjects] = useState<Array<any>>([]);
```

Comment:
https://mantine.dev/hooks/use-list-state/

```suggestion
  const [projects, setProjects] = useListState<Array<any>>([]);
```